tests/test_workshop_v2.py

```diff
@@ -360,7 +360,8 @@ def test_publication_chair(self, client, openreview_client, helpers):
             content={
                 'decision_start_date': start_date.strftime('%Y/%m/%d'),
                 'decision_deadline': due_date.strftime('%Y/%m/%d'),
-                'decision_options': 'Accept, Reject',
+                'decision_options': 'Invite to Venue, Reject',
```

Comment:
Can you add an option `Accept` here and as an `accept_decision_option`? Also, could you add a check for what the venue value should be for every decision option?

Reply:
Also, it looks like the homepage is not showing the tab 'Invite to Venue' with the accepted papers. I think there is a `get_decision_heading_map()` function in helpers that you need to edit. Could you also add this to the test? There is an example here: https://github.com/openreview/openreview-py/blob/c4892b14855691dcafa109b4462e53a7f975286d/tests/test_arr_venue_v2.py#L1899